htdocs/class/xoopsform/renderer/XoopsFormRendererTailwind.php

```
@@ -0,0 +1,738 @@
+     *
+     * @return string rendered form element
+     */
+    public function renderFormColorPicker(XoopsFormColorPicker $element)
+    {
+        if (isset($GLOBALS['xoTheme'])) {
+            $GLOBALS['xoTheme']->addScript('include/spectrum.js');
+            $GLOBALS['xoTheme']->addStylesheet('include/spectrum.css');
+        } else {
+            echo '<script type="text/javascript" src="' . XOOPS_URL . '/include/spectrum.js"></script>';
+            echo '<link rel="stylesheet" type="text/css" href="' . XOOPS_URL . '/include/spectrum.css">';
+        }
```

PHPMD (2.15.0)

[error] The method renderFormColorPicker uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)